onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/MapHelpV2.java

```diff
@@ -205,4 +205,5 @@ public static LatLng getClosestVehicle(ObaTripsForRouteResponse response,
 
         return makeLatLng(closestVehicleLocation);
     }
+
```

Comment:
Could you please remove this white space change to keep the changeset list of files changed clean?